examples/parse_file.rs

```diff
@@ -0,0 +1,45 @@
+extern crate bincode;
+extern crate libc;
+extern crate redland_rs;
+#[macro_use]
+extern crate unwrap;
+
+use libc::{c_char, fclose, fdopen, fgets};
+use redland_rs::{KvStorage, Model, Parser, Serializer, Uri};
+use std::ffi::CStr;
+use std::fs::File;
+use std::io::prelude::*;
+use std::os::unix::io::IntoRawFd;
```

Comment:
This wouldn't work on Windows, but it uses similar traits ([`AsRawHandle`](https://doc.rust-lang.org/std/fs/struct.File.html#impl-AsRawHandle) / `IntoRawHandle`). I'd suggest to handle the platform differences on the library level instead of the app level.